migrate/spring-framework-petclinic-to-15.1.1/migrate/migrate.md

```diff
@@ -27,13 +27,13 @@ Oracle also provides recipes for most common used libraries like Hibernate, ehca
 
 *This lab assumes you have:*
 
-* Basic knowledge of Java, Maven and Git
-* Java 21 or newer (full JDK not a JRE) installed.
+* Basic knowledge of Java, Maven, and Git
+* Java 21 or later (full JDK, not a JRE) installed.
 * [Maven 3.6+](https://maven.apache.org/install.html) installed.
 
 ## Task 1: Run the Maven command to run OpenRewrite
 
-For this example, we will upgrade the Spring Framework PetClinic application to run on WebLogic Server 15.1.1 with JDK 21, including Jakarta EE 9.1, Spring Framework 6.2.x and Hibernate. Will also automatically convert the cache from the deprecated and removed `ehcache 2.0` to Spring Native Cache.
+For this example, we will upgrade the Spring Framework PetClinic application to run on WebLogic Server 15.1.1 with JDK 21, including Jakarta EE 9.1, Spring Framework 6.2.x, and Hibernate. We'll also automatically convert the cache from the deprecated and removed `ehcache 2.0` to Spring Native Cache.
```

Comment:
It's the OpenRewrite that will automatically convert, should be `It'll` instead of `We'll`?

Reply:
Yes, I'll fix that.